app/src/ui/changes/changes-list.tsx

```diff
@@ -704,6 +704,14 @@ export class ChangesList extends React.Component<
     const fileCount = this.props.workingDirectory.files.length
     const filesPlural = fileCount === 1 ? 'file' : 'files'
     const filesDescription = `${fileCount} changed ${filesPlural}`
+
+    const selectedChangeCount = this.props.workingDirectory.files.filter(
+      file => file.selection.getSelectionType() !== DiffSelectionType.None
+    ).length
+    const selectedChangesPlural =
+      selectedChangeCount === 1 ? 'change' : 'changes'
+    const selectedChangesDescription = `${selectedChangeCount} selected ${selectedChangesPlural}`
```

Comment:
Would you mind changing the text slightly here so that instead of `1 change selected` we show `1 changed file selected` to be consistent with the terminology in the header itself?

```suggestion
    const selectedFilesPlural = selectedChangeCount === 1 ? 'file' : 'files'
    const selectedChangesDescription = `${selectedChangeCount} changed ${selectedFilesPlural} selected`
```